utils.py

```diff
@@ -124,3 +125,40 @@ def floatable(str):
     else:
         del parser.rargs[:len(value)]
     setattr(parser.values, option.dest, value)
+
+
+class ColorStr(object):
```

Comment:
the need to be able to colorize the same char referring to different queues or node states. 'a' for alice should be painted in a different colour than 'a' for arty, and i need to be able to attach this information to the string.